Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -168,6 +171,9 @@ function conceptDescription(path, node) {
 
 function getTexts(node, path, unixName) {
 	var conceptDesc = "";
+	var handler = "";
+	var sample = "";
```

Comment:
Variable "handler" is being used in the function later on but the variable "sample" is not being used anywhere. So will delete the variable "sample".